broadcast/index.md

```diff
@@ -31,7 +31,10 @@ and support for Trino. We dive into the details and have a look at a demo.</dd>
 <dd><a href="https://www.linkedin.com/in/manfredmoser">Manfred Moser</a> talks
 about his recent work to improve Trino packaging and demos how to create your 
 own RPM, tarball, and container image.</dd>
-
+<dt>3 Jun 2025: Trino Community Broadcast 75 - Insights from a Norse god</dt>
+<dd><a href="https://www.linkedin.com/in/karsten-j-a7305036/">Karsten Jeschkies</a> 
```

Comment:
Could you link my GitHub profile instead https://github.com/jeschkies/?

Reply:
Done